var/spack/repos/builtin/packages/py-torch/package.py

```diff
@@ -437,6 +437,10 @@ def enable_or_disable(variant, keyword="USE", var=None, newer=False):
             env.set("HIPCUB_PATH", self.spec["hipcub"].prefix)
             env.set("ROCTHRUST_PATH", self.spec["rocthrust"].prefix)
             env.set("ROCTRACER_PATH", self.spec["roctracer-dev"].prefix)
+            env.set("USE_NCCL", True)
```

Comment:
This depends on the value of the `+nccl` variant and is already set below.

Reply:
Removed USE_NCCL.